.circleci/config.yml

```diff
@@ -594,16 +608,15 @@ jobs:
             export IS_PRE_RELEASE=$( if [[ "$BRANCH" =~ "^premainnet-[0-9|.]+$" ]]; then echo true; else echo false; fi )
 
             if [[ $IS_RELEASE  == true ]] || [[ $IS_TEST == true ]] || [[ $IS_PRE_RELEASE == true ]] ; then
-              echo "$DOCKERHUB_PASSWORD" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
               docker/update_or_build.sh -u -p -b ${BRANCH} -n client
               docker/update_or_build.sh -u -p -b ${BRANCH} -n init
               docker/update_or_build.sh -u -p -b ${BRANCH} -n mint
-              docker/update_or_build.sh -u -p -b ${BRANCH} -n safety-rules
               docker/update_or_build.sh -u -p -b ${BRANCH} -n tools
-              docker/update_or_build.sh -u -p -b ${BRANCH} -n validator-dynamic
+              docker/update_or_build.sh -u -p -b ${BRANCH} -n validator
+              docker/update_or_build.sh -u -p -b ${BRANCH} -n validator-tcb
```

Comment:
Why not just use safety-rules so that we don't have to create a mapping in the build script below?